src/constants.js

```diff
@@ -0,0 +1,51 @@
+const clientSideCommands = [
+    "/collapse",
+    "/expand"
+];
+
+const passThroughCommands = [
+	"/as",
+	"/bs",
+	"/cs",
+	"/ho",
+	"/hs",
+	"/join",
+	"/ms",
+	"/ns",
+	"/os",
+	"/rs",
+];
+
+const inputs = [
+	"ban",
+	"ctcp",
+	"msg",
+	"part",
+	"rejoin",
+	"action",
+	"away",
+	"connect",
+	"disconnect",
+	"ignore",
+	"invite",
+	"kick",
+	"mode",
+	"nick",
+	"notice",
+	"query",
+	"quit",
+	"raw",
+	"topic",
+	"list",
+	"whois",
+].reduce(function(plugins, name) {
+	const plugin = require(`./plugins/inputs/${name}`);
```

Comment:
Can we think of a better solution of loading core plugins? I don't like this being in a file called constants

Reply:
We can just have this file list the core commands and we can import the files in another folder. I guess the thing is, it makes perfect sense to have the list of command names in here, but somewhere else (I'll have a think later) can import them specifically?

Reply:
So, realisticall,y this isn't really a constants file. So I've renamed it and moved it into the `inputs` folder, as that's all it's referring to.